R/1-4-0safelearn.R

```diff
@@ -34,16 +34,20 @@
 #' @importFrom purrr map
 #' @keywords internal
 #' @noRd
+
 safelearn <- function(
     c.vec,
     n,
     q,
     cost,
     M,
-    group_name,
+    group_name_list,
     dif_lip_output,
     cross_fit_output,
     trace) {
+
+  temp_reg_df <- data.frame(M = numeric(), C = numeric(), group = integer(), c_alt = numeric(), temp_reg = numeric(), stringsAsFactors = FALSE)
```

Comment:
Consider applying the [styler](https://cran.r-project.org/web/packages/styler/index.html). 

```r
temp_reg_df <- data.frame(
  M = numeric(),
  C = numeric(),
  group = integer(),
  c_alt = numeric(),
  temp_reg = numeric(),
  stringsAsFactors = FALSE
)
```